docs/sources/menu.yaml

```diff
@@ -0,0 +1,312 @@
+  - name: Screencasts
+    link: /tutorials/screencasts/
+- name: Features
+  link: /features/
+  children:
+  - name: Panels
+    link: /features/panels/
+    children:
+    - link: /features/panels/graph/
+      name: Graph
+    - link: /features/panels/logs/
+      name: Logs
+    - link: /features/panels/singlestat/
+      name: Singlestat
+    - link: /features/panels/table_panel/
+      name: Table
+    - link: /features/panels/heatmap/
+      name: Heatmap
+    - link: /features/panels/alertlist/
+      name: Alert list
+    - link: /features/panels/dashlist/
+      name: Dashboard list
+    - link: /features/panels/text/
+      name: Text
+  - name: Dashboard Features
+    link: /features/dashboard/
+    children:
+    - link: /reference/templating/
+      name: Variables
+    - link: /reference/annotations/
+      name: Annotations
+    - link: /reference/dashboard_folders/
+      name: Folders
+    - link: /reference/playlist/
+      name: Playlist
+    - link: /reference/search/
+      name: Search
+    - link: /reference/share_dashboard/
+      name: Sharing a Dashboard
+    - link: /reference/share_panel/
+      name: Sharing a Panel
+    - link: /reference/timerange/
+      name: Time Range
+    - link: /reference/export_import/
+      name: Export and Import
+    - link: /reference/scripting/
+      name: Scripted Dashboards
+    - link: /reference/dashboard_history/
+      name: Dashboard Version History
+    - link: /reference/dashboard/
+      name: JSON Model
+  - name: Data Sources
+    link: /features/datasources/
+    children:
+    - link: /features/datasources/graphite/
+      name: Graphite
+    - link: /features/datasources/prometheus/
+      name: Prometheus
+    - link: /features/datasources/influxdb/
+      name: InfluxDB
+    - link: /features/datasources/elasticsearch/
+      name: Elasticsearch
+    - link: /features/datasources/stackdriver/
+      name: Google Stackdriver
+    - link: /features/datasources/cloudwatch/
+      name: AWS Cloudwatch
+    - link: /features/datasources/azuremonitor/
+      name: Azure Monitor
+    - link: /features/datasources/loki/
+      name: Loki
+    - link: /features/datasources/mysql/
+      name: MySQL
+    - link: /features/datasources/postgres/
+      name: PostgreSQL
+    - link: /features/datasources/mssql/
+      name: Microsoft SQL Server
+    - link: /features/datasources/opentsdb/
+      name: OpenTSDB
+    - link: /features/datasources/mixed/
+      name: MixedData
+    - link: /features/datasources/testdata/
+      name: TestData
+  - name: Explore
+    link: /features/explore/
+  - name: Alerting
+    link: /alerting/
+    children:
+    - link: /alerting/rules/
+      name: Engine and Rules
+    - link: /alerting/metrics/
+      name: Metrics
+    - link: /alerting/notifications/
+      name: Notifications
+  - name: Keyboard Shortcuts
+    link: /features/shortcuts/
+- name: What's New In Grafana
+  link: /whatsnew/
+  children:
+  - name: Version 6.4
```

Comment:
6.5 is missing. You sure you've based your changes on the latest commit in master?

Reply:
Yes, forgot to update the file.